routes.jsx

```diff
@@ -15,21 +15,38 @@ import Navbar from './components/navbar/navbar.jsx';
 import Footer from './components/footer/footer.jsx';
 
 const App = React.createClass({
+  getInitialState() {
+    return {
```

Comment:
the code that is currently in componentDidMount should be here, so that the `suppressSplashScreen` value is correct immediately (remember, localStorage exists well before the app does, so don't need to "wait" for the app, the value exists irrespective of the app)